api/src/test/resources/io/strimzi/api/kafka/model/Kafka-with-invalid-jmx-authentication.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: kafka.strimzi.io/v1beta1
+kind: Kafka
+metadata:
+  name: strimzi-ephemeral
+spec:
+  kafka:
+    replicas: 3
+    storage:
+      type: persistent-claim
+      size: 500Gi
+    listeners:
+      plain: {}
+      tls: {}
+    jmxOptions:
+      authentication:
+        type: "not-right"
+  zookeeper:
+    replicas: 3
+    storage:
+      type: ephemeral
+  maintenanceTimeWindows:
+    - first window
+    - second window
```

Comment:
I think you can get rid of this `maintenanceTimeWindows` section for this kind of test.